model/messaging/spans.yaml

```diff
@@ -118,6 +118,22 @@ groups:
           Message [correlation Id](https://www.rabbitmq.com/tutorials/tutorial-six-java#correlation-id) property.
       - ref: messaging.message.body.size
         requirement_level: opt_in
+      - ref: messaging.destination.name
+        note: |
+          In RabbitMQ, the destination is defined by an *exchange*, a *routing key* and for consumers, a *queue*.
+
+          `messaging.destination.name` SHOULD be set to:
+
+          - **Producers**: `{exchange}:{routing key}` when both values are present and non-empty.
+          Otherwise: `amq.default` when the default exchange is used and no routing key is provided
+
+          - **Consumers**: `{exchange}:{routing key}:{queue}` when all values are present and non-empty.
+          If any has an empty value (e.g., the default exchange is used) it SHOULD be omitted.
+          For cases when `{routing key}` and `{queue}` are equal, only one of them SHOULD
+          be used, e.g., `{exchange}:{routing key}`.
+        examples: ['direct_logs:warning', 'logs']
+        requirement_level:
+          conditionally_required: If span describes operation on a single message or if the value applies to all messages in the batch.
```

Comment:
Does rabbit support batching at all? I don't think it does. Wonder if we can just say that it's `required`  (or `conditionally_required: if available`)

Reply:
Yes, true, there's pre-fetch but we explicitly say to not create receive/process span for those so I think we can remove this from Rabbit. 

I'm leaning towards just marking it as required, as I don't see a case when it's not available.. _something_ is always available, (like even when using the default exchange, or anonymous queues). [bde99d0](https://github.com/open-telemetry/semantic-conventions/pull/1531/commits/bde99d03a24315db9777f533fe7d0832accb53d5)